### ISSUE TITLE
Incapsulate network events

### ISSUE THREAD

**Issue by surabujin** [telstra/open-kilda#686]
Problem definition
------------------

Floodlight provide some set of network events (switch up/down, port up/down, isl ping/lost). The order of this events and their values build Kilda's view of network.

To make this network view solid - kilda must interpret this events into it's own set of management signals. The complex part in this process is complex behaviour based on network events order (we should keep in mind that kafka queue can change events order during delivery). The more mess in process add spread nature of network event processing ("raw" events processed at least in "event topoloy", TE, CT). So all this spread components must implement complex logic of network events relationship (isl from ports, ports from switch etc). Also they should take into account each other.

In other words it complex and mostly unmanageable system. With extremely high price on adding new features/fixing problems.

Proposition
-----------
We must have only one component that handle(receive) raw network events. This component must sort this events into correct order(eliminate kafka's reordering), keep track of events relationships. And as results produce set of our internal events - (create/remove/deactivate isl, create/remove/deactivate switch, evacuate flow etc...)

In this case all raw events logic will be isolated in one component. And all other component will receive simple not connected to each other events.
